deployments/liqo/README.md

```diff
@@ -24,6 +24,12 @@
 | auth.service.annotations | object | `{}` | auth service annotations |
 | auth.service.type | string | `"NodePort"` | The type of service used to expose the Authentication Service If you are exposing this service with an Ingress consider to change it to ClusterIP, otherwise if you plan to use liqo over the Internet consider to change this field to "LoadBalancer". See https://doc.liqo.io/user/install/pre-install/ for more details. |
 | auth.tls | bool | `true` | Enable TLS for the Authentication Service Pod (using a self-signed certificate). If you are exposing this service with an Ingress consider to disable it or add the appropriate annotations to the Ingress resource. |
+| capsule | object | `{"fullnameOverride":"capsule","install":true,"manager":{"image":{"tag":"v0.1.0-rc2"}}}` | capsule subchart configuration |
+| capsule.fullnameOverride | string | `"capsule"` | override the fullname to fix naming problems |
+| capsule.install | bool | `true` | liqo needs capsule to work properly, but you can use your already deployed capsule installation |
+| capsule.manager | object | `{"image":{"tag":"v0.1.0-rc2"}}` | capsule manager configuration |
+| capsule.manager.image | object | `{"tag":"v0.1.0-rc2"}` | manager image settings |
+| capsule.manager.image.tag | string | `"v0.1.0-rc2"` | manager image tag |
```

Comment:
Would double-check the generated output, as it seems strange to me (probably some comments should not have the `--` marker.